pkg/web/templates/components/add_apikey.html

```diff
@@ -0,0 +1,130 @@
+            <label for="pki_manage">pki:manage</label>
+            <input type="checkbox" id="pki_manage" name="permissions[]" value="pki:manage" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="pki_view">pki:view</label>
+            <input type="checkbox" id="pki_view" name="permissions[]" value="pki:view" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="pki_delete">pki:delete</label>
+            <input type="checkbox" id="pki_delete" name="permissions[]" value="pki:delete" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="transactions_manage">transactions:manage</label>
+            <input type="checkbox" id="transactions_manage" name="permissions[]" value="transactions:manage" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="transactions_view">transactions:view</label>
+            <input type="checkbox" id="transactions_view" name="permissions[]" value="transactions:view" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="travelrule:manage">travelrule:manage</label>
+            <input type="checkbox" id="travelrule:manage" name="permissions[]" value="travelrule:manage" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="travelrule:view">travelrule:view</label>
+            <input type="checkbox" id="travelrule:view" name="permissions[]" value="travelrule:view" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="travelrule:delete">travelrule:delete</label>
+            <input type="checkbox" id="travelrule:delete" name="permissions[]" value="travelrule:delete" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="users:manage">users:manage</label>
+            <input type="checkbox" id="users:manage" name="permissions[]" value="users:manage" class="checkbox-style" />
+          </div>
+          <div class="my-2 grid grid-cols-2 items-center">
+            <label for="users:view">users:view</label>
+            <input type="checkbox" id="users:view" name="permissions[]" value="users:view" class="checkbox-style" />
+          </div>
+        </section>
+        <div class="flex justify-center">
+          <button id="acct-submit-bttn" class="submit-btn w-44 btn bg-primary font-semibold text-lg text-white hover:bg-primary/90">
+            <span class="submit-btn-text">Create</span>
+            <span id="loader" class="htmx-indicator loading loading-spinner loading-md"></span>
+          </button>
```

Comment:
I think that's all the permissions! The list is here:

https://github.com/trisacrypto/envoy/blob/main/pkg/web/auth/permissions/permissions.go#L45-L54

There should be 18 including "unknown" which can be omitted from this list, so 17.